app/data/1_0102.json

```diff
@@ -171,7 +190,13 @@
                                             }
                                         }
                                     ],
-                                    "description": "Round your figures to the nearest whole pound (£). Even if your figures are zero, please still complete.<div><br/> <h4>Include</h4><ul> <li>VAT</li></ul></div>",
+                                    "description": "Round your figures to the nearest whole pound (£). Even if your figures are zero, please still complete.",
```

Comment:
I realise this isn't a change we introduced here but we should be consistent with how we apply the paragraphs in the description as this doesn't have a wrapping `<p>` tag but others do.